src/apps/lwaftr/binding_table.lua

```diff
@@ -32,33 +32,72 @@ end
 
 local machine_friendly_binding_table
 
--- b4_v6 is for the B4, br_v6 is for the border router (lwAFTR)
+--[[
+b4_v6 is for the B4, br_v6 is for the border router (lwAFTR)
+
+Binding table format:
+   - Entry: {b4_v6, v4, { psid_info }, br_v6}
+   - PSID Info: {reserved_ports_bit_count=bits, psid_len=bits, shift=bits}
```

Comment:
We're not storing all three. Please have the comment reflect the two we are storing, and mention that the third is computed.